docs/using_tilt_extensions.md

```diff
@@ -15,9 +15,17 @@ Once you've found an extension, using it couldn't be easier! Say we wanted to us
 load('ext://hello_world', 'hi')
 ```
 
-`ext://hello_world` is resolved to https://github.com/windmilleng/tilt-extensions/blob/master/hello_world/Tiltfile by Tilt. When the extension is loaded in to your project for the first time Tilt writes the Tiltfile in to the `tilt_modules` directory in your project. So in this case Tilt would write the contents of `ext://hello_world` to `tilt_modules/hello_world`.
+`ext://hello_world` is resolved to https://github.com/windmilleng/tilt-extensions/blob/master/hello_world/Tiltfile by Tilt. When the extension is loaded into your project for the first time, Tilt copies the remote Tiltfile into the `tilt_modules` directory of local your project. So in this case Tilt would write the contents of `ext://hello_world` to `tilt_modules/hello_world`.
+
+`hi` is the name of the function defined in [hello_world/Tiltfile](https://github.com/windmilleng/tilt-extensions/blob/master/hello_world/Tiltfile) and now made available to your local project Tiltfile. Simply call `hi()` anywhere to execute it, printing "Hello world!".
 
 The `tilt_modules` directory should be committed to your repo (**not** `gitignore`'d) to ensure that your teammates don't have to download it in the future, and to ensure that everyone on your team is using the same version of the extension.
 
+## Changing an Extension
+
+We don't recommend changing extensions directly in `tilt_modules`. If you want to customize an extension you are already using, consider [contributing a new extension](writing_tilt_extensions.html).
+
+Tilt doesn't support versioning of extensions currently. If you are interested in versioning, [let us know](https://tilt.dev/contact).
```

Comment:
This doesn't seem like it fits under "Changing an Extension". How about at the end of "Using an Extension"?

Reply:
Makes sense. Moved to end of "Using an Extension".